pxr/imaging/hio/glslfxConfig.cpp

```diff
@@ -438,7 +438,7 @@ HioGlslfxConfig::_GetParameters(VtDictionary const & dict,
 
     const VtDictionary& paramsDict = params.UncheckedGet<VtDictionary>();
     // pre-process the paramsDict in order to get the merged ordering
-    for (const std::pair<std::string, VtValue>& p : paramsDict) {
+    for (const std::pair<std::string, VtValue> p : paramsDict) {
```

Comment:
You should be able to avoid the copy by making `p` `const auto&` or `const std::pair<const std::string, VtValue>&`.